__tests__/app.test.js

```diff
@@ -72,6 +72,23 @@ describe("/api/", () => {
     });
   });
 });
+describe("/api/users", () => {
+  test("200: returns an array of user objects with the correct keys", () => {
```

Comment:
You could leave it at 'returns an array of user objects'.  We can assume they will be correct.